tests/test_pe_mailer.py

```diff
@@ -0,0 +1,101 @@
+"""File to be used to create test for pe-mailer module."""
```

Comment:
Prefer simpler wording:
```suggestion
"""Tests for the pe-mailer module."""
```